include/range/v3/detail/config.hpp

```diff
@@ -389,6 +389,14 @@ namespace ranges
 #define RANGES_RE_ENABLE_WARNINGS
 #endif
 
+
+#if __has_attribute(no_sanitize)
+#define RANGES_INTENDED_UNSIGNED_OVERFLOW \
```

Comment:
The pedant in me needs to point out that the word overflow describes exactly what happens. Exceeding the maximum representable value is an overflow. The difference is whether the overflow has defined behavior or not.

Just my two cents.